release_notes.md

```diff
@@ -1,21 +1,10 @@
 # What's New
 
-## Introducing Top SOC Questions
-
-Edit or disable existing prompts or add new prompts based on your security posture.
-
-- You can now customize your interactions with the LLM by providing specific details and guidelines for your chats. Define your SOC questions to customize your LLM interactions.
-
-- Specific modules now have pre-defined questions. For example, under the **Alerts** module, questions include:
-
-    - Get Alert Summary
-    - Generate Response Recommendations
-    - Generate Investigation Report
-    - MITRE ATT&CK Insight
-    
-    Similarly, for the **Incident** module, questions may be:
-    
-    - Get Incident Summary
-    - Generate Response Recommendations
-    - Generate Investigation Report
-    - MITRE ATT&CK Insight
+- The AI Assistant can understand and respond to context sensitive conversations and perform given SOC tasks
+    - Through enhanced comprehension, the bot can navigate to modules, its records, and playbook logs
```

Comment:
there are more such use-cases like alert investigation, remediation using block ioc, etc.

Reply:
escalate alert to an incident, setup warroom, etc.